scripts/svgGraphs.js

```diff
@@ -154,140 +154,142 @@ const {barLineGraph, stackedAreaGraph, pieGraph} = (function () {
-            .call(g => g.select(".domain").remove())
-            .call(g => g.append("text")
-                .attr("x", -barMargin.left)
-                .attr("y", 10)
-                .attr("fill", "currentColor")
-                .attr("text-anchor", "start"));
-
-        const xValues = data[0][1].x;
-        const x = d3.scaleBand()
-            .domain(d3.range(xValues.length))
-            .range([barMargin.left, width - barMargin.right])
-            .padding(0.1);
-
-        if (minYValue < 0) {
-            svg.append("g")
-                .append("line")
-                .attr("x1", barMargin.left)
-                .attr("x2", width - barMargin.right)
-                .attr("y1", y(0))
-                .attr("y2", y(0))
-                .attr("style", "stroke-width: .5px")
-                .attr("stroke", "currentColor");
-        }
-
-        let xGroupOffset;
-        if (config.groupBars) {
-            xGroupOffset = d3.scaleBand()
-                .domain(data.map(([title]) => title))
-                .rangeRound([0, x.bandwidth()])
-                .padding(0.05);
-        } else {
-            xGroupOffset = () => 0;
-        }
-
-        const interval = Math.ceil(xValues.length / 40);
-        const xAxis = g => g
-            .attr("transform", `translate(0,${height - barMargin.bottom})`)
-            .call(d3.axisBottom(x)
-                .tickValues(x.domain().filter((_, i) => i % interval === 0))
-                .tickFormat(i => xValues[i])
-                .tickSizeOuter(0));
-
-        const renderSeries = (seriesTitle, seriesData, seriesIndex, seriesToRender) => {
-            const data = seriesData.x.reduce((acc, value, index) => {
-                acc.push({
-                    [seriesTitle]: value,
-                    value: Number(seriesData.y[index])
-                });
-                return acc;
-            }, []);
-
-            if (seriesToRender === "line" && seriesData.type === "line") {
-                const line = d3.line()
-                    .x((d, i) => x(i))
-                    .y(({value}) => value < 0 ? y(0) : y(value));
-
-                svg.append("path")
-                    .datum(data)
-                    .attr("fill", "none")
-                    .attr("class", `graph-series-stroke-${seriesIndex + 1}`)
-                    .attr("stroke-width", 1.5)
-                    .attr("stroke-linejoin", "round")
-                    .attr("stroke-linecap", "round")
-                    .attr("d", line);
-            } else if (seriesToRender === "bar" && (seriesData.type === undefined || seriesData.type === "bar")) {
+        if (data.length > 0) {
+            const tooltip = d3.select("#" + id)
+                .append("div")
+                .attr("class", "graph-tooltip")
+                .style("display", "none");
+
+            const mouseAction = (event, {value}) => {
+                tooltip.style("display", "")
+                    .text(d3.format('d')(value))
```

Comment:
I also changed format of tooltip to avoid .666666666666666666 causing tooltip to go off screen some or get clipped in iframe.